wackywebm.js

```diff
@@ -223,29 +225,45 @@ function ffmpegErrorHandler(e) {
 	)
 }
 
+// Creates a new Promise to allow the script to wait for the
+// YouTube video to finish downloading and saves it in the
+// temporarly download folder
+async function downloadYoutubeVideo(url, newFilePath) {
+	await fs.promises.mkdir(workLocations.tempYoutubeDownload, { recursive: true })
+
+	return new Promise((resolve) => {
```

Comment:
can we get graceful error handling here? just reject the promise instead of setting the program on fire if something goes wrong

Reply:
I've done this but as of 10 hours ago, ytdl-core has some issues for which they haven't released fixes for yet. While I'm sure they will come up with one, I think I'll try to do this same thing with pure ffmpeg and create another PR, so feel free to close this one.